App/Redux/LogInReducer.ts

```diff
@@ -0,0 +1,76 @@
+/// <reference path="../../typings/tsd.d.ts" />
+import { ISettings, IUserProfile, ISettingsInfo, ISettingsLists} from './LoginActions';
+
+export interface ISettingsState{
+  account: string;
+  project: string;
+  team: string;
+}
+
+const initialState: ISettingsState = {
+    account: 'acc1',
+    project: 'proj1',
+    team: 'team1',
+};
+
+export function updateDefaultState(state: ISettingsState = initialState, action: ISettings): ISettingsState {
+  switch (action.type) {
+    case 'DEFAULT_SETTINGS':
+      return Object.assign({}, state, { account: action.account, project: action.project, team: action.team});
+    default:
+      return state;
+  }
+}
+
+export interface IAccountLists {
+  accountList: ISettingsInfo[];
+}
+
+export interface ITeamLists {
+  teamList: ISettingsInfo[];
+}
+export interface IProjectLists {
+  projectList: ISettingsInfo[];
+}
+
+export interface ICurrentLists {
+  accountList: ISettingsInfo[];
```

Comment:
Why are you not using the above interfaces that you defined? If they are not needed then remove them.